packages/babel-jest/src/__tests__/index.ts

```diff
@@ -20,7 +23,7 @@ jest.mock('../loadBabelConfig', () => {
   };
 });
 
-const defaultBabelJestTransformer = babelJest.createTransformer(null);
+const defaultBabelJestTransformer = babelJest.createTransformer();
```

Comment:
`null` is not allowed, but could be `undefined`. Perhaps it is fine to call it with no args:

https://github.com/facebook/jest/blob/eafabf99e64ac03f0af40ca26f24ebbb0f401c77/packages/babel-jest/src/index.ts#L151-L155